src/test/elements/successfactors/user-accounts.js

```diff
@@ -21,8 +22,7 @@ suite.forElement('humancapital', 'user-accounts', null, (test) => {
       .then(r => cloud.get(`${test.api}/${id}`))
       .then(r => cloud.put(`${test.api}/${id}`, userAccountsUpdatePayload));
   });
-  
-  //skipping due to EL-1076
+
```

Comment:
Go ahead and leave this comment 

```suggestion
//skipping due to EL-1076
```